MESC_Common/Src/MESCsin_lut.c

```diff
@@ -1,12 +1,28 @@
 /*
- * sin_cos.h
+ **
+ ******************************************************************************
+ * @file           : MESCmotor_state.c
+ * @brief          : Code for motor state machine
+ ******************************************************************************
+ * @attention
  *
- *  Created on: Dec 28, 2020
- *      Author: salavat.magazov
+ * <h2><center>&copy; Copyright (c) 2020 David Molony.
+ * All rights reserved.</center></h2>
+ *
+ * This software component is licensed under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+
+ * MESCsin_lut.c
+ *
+ *  Created on: 25 Jul 2020
+ *      Author: David Molony
```

Comment:
This description is wrong and should be updated accordingly

Reply:
see new commit